doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -7,220 +7,125 @@ NumPy for MATLAB users
-      cast as single-column or single-row matrices.
-   -  ``<:(`` Since ``array`` is the default in NumPy, some functions may
-      return an ``array`` even if you give them a ``matrix`` as an
-      argument. This shouldn't happen with NumPy functions (if it does
-      it's a bug), but 3rd party code based on NumPy may not honor type
-      preservation like NumPy does.
-   -  ``:)`` ``A*B`` is matrix multiplication, so it looks just like you write
-      it in linear algebra (For Python >= 3.5 plain arrays have the same
-      convenience with the ``@`` operator).
-   -  ``<:(`` Element-wise multiplication requires calling a function,
-      ``multiply(A,B)``.
-   -  ``<:(`` The use of operator overloading is a bit illogical: ``*``
-      does not work element-wise but ``/`` does.
-   -  Interaction with ``scipy.sparse`` is a bit cleaner.
-
-The ``array`` is thus much more advisable to use.  Indeed, we intend to
-deprecate ``matrix`` eventually.
 
-Table of Rough MATLAB-NumPy Equivalents
+Rough equivalents
 =======================================
 
 The table below gives rough equivalents for some common MATLAB
-expressions. **These are not exact equivalents**, but rather should be
-taken as hints to get you going in the right direction. For more detail
-read the built-in documentation on the NumPy functions.
+expressions. These are similar expressions, not equivalents. For
+details, see the `documentation
+<https://numpy.org/doc/stable/reference/routines.html>`_. 
 
 In the table below, it is assumed that you have executed the following
 commands in Python:
 
 ::
 
-    from numpy import *
-    import scipy.linalg
+    import numpy as np
+    from scipy import io, integrate, linalg, signal
+    from scipy.sparse.linalg import eigs
 
 Also assume below that if the Notes talk about "matrix" that the
 arguments are two-dimensional entities.
 
-General Purpose Equivalents
+General purpose equivalents
 ---------------------------
 
 .. list-table::
    :header-rows: 1
 
    * - **MATLAB**
-     - **numpy**
+     - **NumPy**
      - **Notes**
 
    * - ``help func``
-     - ``info(func)`` or ``help(func)`` or ``func?`` (in Ipython)
+     - ``info(func)`` or ``help(func)`` or ``func?`` (in IPython)
      - get help on the function *func*
 
    * - ``which func``
-     - `see note HELP <numpy-for-matlab-users.notes>`__
+     - `see note HELP <numpy-for-matlab-users.notes>`_
      - find out where *func* is defined
 
    * - ``type func``
-     - ``source(func)`` or ``func??`` (in Ipython)
+     - ``source(func)`` or ``func??`` (in IPython)
      - print source for *func* (if not a native function)
 
+   * - ``% comment``
+     - ``# comment``
+     - comment a line of code with the text ``comment``
+
+   * - ::
+
+         for i=1:3
+             fprintf('%i\n',i)
+         end
+
+     - ::
+
+         for i in range(1, 4):
+            print(i)
+
+     - use a for-loop to print the numbers 1, 2, and 3 using `range
```

Comment:
```suggestion
     - use a for-loop to print the numbers 1, 2, and 3 using :func:`range`
```
And delete the url below.

Reply:
> And delete the url below.

my html did not generate a link with this change. Is there a way to specify Python instead of NumPy function?